include/megaapi_impl.h

```diff
@@ -1673,6 +1675,8 @@ class MegaApiImpl : public MegaApp
         MegaClient *getMegaClient();
         static FileFingerprint *getFileFingerprintInternal(const char *fingerprint);
 
+        virtual std::string getFileAttribute(MegaHandle handle);
```

Comment:
If the corresponding function in the public object is changed. Change this one accordingly.